### DISCUSSION TITLE
Atmega328 / SX1278 Timer Interrupt

### DISCUSSION THREAD

**Discussion by lw2dtz** [jgromes/RadioLib#840] [Q&A]
I'm trying to use timer interrupts using the TimerOne library using an atmega328p and SX1278 but when I activate the interrupt seems to get stuck the controller. Activate debug mode but only meaningless characters appear. I leave the sketch here. 
Any ideas ?
[AFSK_SSTV_A.txt](https://github.com/jgromes/RadioLib/files/12778866/AFSK_SSTV_A.txt)

**Comment by jgromes** [comment]
In which version of the library is this behavior present? If it's the latest, then I'm afraid it might be due to the size no longer being able to fit on low-end devices such as ATmega328P. What is the reported size after compilation?
